db/system_keyspace.cc

```diff
@@ -264,6 +264,9 @@ schema_ptr system_keyspace::topology() {
             .with_column("new_cdc_generation_data_uuid", timeuuid_type, column_kind::static_column)
             .with_column("new_keyspace_rf_change_ks_name", utf8_type, column_kind::static_column)
             .with_column("new_keyspace_rf_change_data", map_type_impl::get_instance(utf8_type, utf8_type, false), column_kind::static_column)
+            .with_column("truncate_table_ks_name", utf8_type, column_kind::static_column)
+            .with_column("truncate_table_cf_name", utf8_type, column_kind::static_column)
+            .with_column("truncate_table_timeout_ms", long_type, column_kind::static_column)
```

Comment:
you have to update system topology desc in docs/dev/topology-over-raft.md

Reply:
Updated in v6